tests/unit/controllers/account/billing-info/invoices-test.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Controller | account/billing-info/invoices', function(hooks) {
```

Comment:
this test is failing in travis

Reply:
@Anupam-dagar Some other test seems to be failing :(